.travis.yml

```diff
@@ -4,11 +4,12 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 install:
   - composer self-update
-  - composer global require "fxp/composer-asset-plugin:1.1.2"
+  - composer global require "fxp/composer-asset-plugin:~1.1.1"
```

Comment:
I set minimum version to 1.1.1. On 1.1.2 I had an error, maybe composer broke api again. https://github.com/yiisoft/yii2/issues/10554#issuecomment-171623038